code-list-scripts/gremlin/census_2021_year_arrival_uk (1).grm

```diff
@@ -0,0 +1,112 @@
+g.V().hasId('_code_list_census_2021_year_arrival_uk_one-off').

```

Comment:
everycode list id has `hypen` but here is underscore and the title has (1) in it.  Not sure if this is intended

Reply:
good spot, ive amended the file name

Reply:
The list id property has  'underscore' instead of 'hypen' is that ok?